### PR TITLE
Generate presto-internal.thrift file

### PR DESCRIPTION
**Pull request by jbroll**
Test plan - 

run 
```
mvn process-classes -DskipTests -T1C
```

inspect presto-thrift-spec/target/presto-internal-0.247-SNAPSHOT.thrift

```
mvn clean install -DskipTests -T1C -pl '!presto-docs,!presto-server-rpm'
jroll@fb : ls -l ~/.m2/repository/com/facebook/presto/presto-thrift-spec/0.247-SNAPSHOT/presto-thrift-spec-0.247-SNAPSHOT.thrift
-rw-r--r--  1 jroll  staff  1371 Jan 22 09:29 /Users/jroll/.m2/repository/com/facebook/presto/presto-thrift-spec/0.247-SNAPSHOT/presto-thrift-spec-0.247-SNAPSHOT.thrift
```
== RELEASE NOTES ==

General Changes
* Add maven plugin for drift to generate the Thrift spec for a list of classes, current class list is **TaskStatus**

### CODE REVIEW

File: presto-main/pom.xml [hunk 1]
```diff
@@ -468,6 +468,47 @@
                     </excludes>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>com.facebook.drift</groupId>
+                <artifactId>drift-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>execution1</id>
```
Comment (ajaygeorge) [inline presto-main/pom.xml:476]: is `execution`  already used in the context which forces us to append `1`

File: presto-main/pom.xml [hunk 2]
```diff
@@ -468,6 +468,47 @@
...
+                        <goals>
+                            <goal>generate-thrift-idl</goal>
+                        </goals>
+                        <phase>process-classes</phase>
+                        <configuration>
+                            <classes>
+                                <class>com.facebook.presto.execution.TaskStatus</class>
+                            </classes>
+                            <recursive>true</recursive>
+                            <outputFile>target/presto-internal-${project.version}.thrift</outputFile>
```
Comment (ajaygeorge) [inline presto-main/pom.xml:486]: how about keep the file in a new folder instead of creating it under the parent folder?

**Comment by tdcmeehan** [comment]
I imagine that whoever needs this artifact doesn't need the contents in `presto-main`.  Does it make sense to put it in its own module, whose sole artifact is the Thrift IDL?

File: presto-main/pom.xml (hunk 2 above)
Comment (jbroll) [reply presto-main/pom.xml]: Moved to a new module as suggested originally.

**Review by ajaygeorge — APPROVED** [review]
LGTM

**Pull request merged**